mbf_costmap_nav/src/nav_core_wrapper/wrapper_recovery_behavior.cpp

```diff
@@ -59,7 +59,8 @@ uint32_t WrapperRecoveryBehavior::runBehavior(std::string& message)
 
 bool WrapperRecoveryBehavior::cancel()
 {
-  return false;
+  nav_core_plugin_->cancel();
```

Comment:
maybe return the bool from the cancel()?

Reply:
sure.